Makefile

```diff
@@ -513,7 +513,7 @@ test-ci-native: | test/addons/.buildstamp test/js-native-api/.buildstamp test/no
 .PHONY: test-ci-js
 # This target should not use a native compiler at all
 # Related CI job: node-test-commit-arm-fanned
-test-ci-js: | clear-stalled
+test-ci-js: | clear-stalled bench-addons-build
```

Comment:
The comment for this target says it should not use a native compiler, which building an addon does.

Reply:
Oooh, good catch. Fixing.